### ISSUE TITLE
Week 11 - Review Ticket (William Li)

### ISSUE THREAD

**Issue by willili-ljw** [aidanywu/furious_five#129]
# Self Review: 8.9/10

# Peer Review: 8.5/10

Total Score: 8.9/10

Individual/Pair Evaluation Create Task Project
- Wiki/Jekyll page with design ideas and usage
Score: 1/1
- Create Performance Task PBL feature
Score: 1/1
- Final committed program code showing algorithms written in JS/Python
Score: 1/1
- Written responses to all the prompts in the performance task
Score: .9/1; written responses on google docs instead of on wiki
- 1min video that meets CB create task requirements
Score: 1/1

Team Final Grading Considerations: 4/5

 -  PBL Github pages. Jekyll, WIKI, live review focus
 - Blueprint shown in about me pages 
 - Crud Async Operation 
 - Individual PBL MVC feature/commits is shown in the video and on the insights 
 -  Runtime Features shown in 1 min video
 - Highlight of best features
 - CRUD/Search on project database tables, easy to identify from UI (merging error and will talk to teacher)
 - MVC
 - Google Search or Google translate
 - Scrum Board 

4/5, video did not have audio, lacked in quality, and did not show async CRUD.

**Comment by AD1616** [comment]
Sahil Review of William:

**Individual Evaluation: 4.8/5** 
- Wiki page with design ideas and usage [here](https://github.com/aidanywu/furious_five/wiki/Create-Task-Details#william-li) 1/1
- Timer feature in create task related to PBL which is a study tool. 1/1
- Program code has algorithms written in javascript. 1/1
- Written responses to all prompts in the create performance task is there; however it is done on google docs and not on a wiki. 0.9/1
- Video clearly shows functionality. However, some subtitles would have been nice to clarify what the different boxes were doing when setting the lengths for the timer. 0.9/1.

**Team Review: 3.7/5**
- Home page briefly describes PBL theme. However, this is very outdated and does not broadly talk about the theme and only gives a few specific features. Brainwrite has more ideas, but no overarching theme page. 0.4/0.5
- Used bootstrap through the dark mode feature. Did not use sassy. 0.4/0.5.
- Used blueprints in the code. No wiki but illustrated in code. 0.4/5.
- No CRUD async. 0/0.5
- Individual MVC feature is there through the calendar which uses local storage data for the model, html for the view, and control to add assignments. Has many commits in javascript algorithms. 0.5/0.5
- Google search is implemented. 0.5/0.5
- Completed tasks are shown on the scrumboard. Tangibles are not on the tasks, but they can be seen in commits; scrumboard is updated with closed issues and is organized well. 0.4/0.5.
- A search function is shown, but it is not crud. 0.1/0.5
- Best features are highlighted. 0.5/0.5
- Deployment policies are on github pages. 
- MVC is shown at the beginning of the video. 0.5/0.5
- Video is a bit hard to follow because there is no narration; William walked me through it. I identified MVC, google search, and scrumboard easily seen and done well. 

Overall score: 8.5/10
